src/Nethermind/Nethermind.JsonRpc/ResultWrapper.cs

```diff
@@ -43,7 +43,7 @@ public static ResultWrapper<T> Fail(string error)
         
         public static ResultWrapper<T> Fail(Exception e)
         {
-            return new() { Result = Result.Fail(e.ToString()), ErrorCode = ErrorCodes.InternalError};
+            return new() { Result = Result.Fail(e.Message), ErrorCode = ErrorCodes.InternalError};
```

Comment:
Any particular reason for this? This potentially will make harder to investigate users issues